packages/storybook/src/generators/configuration/configuration.ts

```diff
@@ -132,7 +132,8 @@ export async function configurationGenerator(
     }
   }
 
-  const initTask = initGenerator(tree, {
+  const initTask = await initGenerator(tree, {
+    js: schema.js,
```

Comment:
this js passed to initGenerator is just to determine whether the typescript package needs to be installed or not. it does not create any files.

Reply:
I understand, but I think it would be cleaner for it not to be here... If you think it's better to be there rather than `configuration` generator, then ok...